contrib/libhdfs3-cmake/CMakeLists.txt

```diff
@@ -129,3 +135,7 @@ if (TARGET OpenSSL::SSL)
 endif()
 
 add_library(ch_contrib::hdfs ALIAS _hdfs3)
+
+# target 
+add_executable(perf_checksum ${HDFS3_SOURCE_DIR}/common/perf_checksum.cpp)
```

Comment:
Benchmarks are without doubt important and building perf_checksum does not take long. I nevertheless doubt that we should build the benchmark by default. Perhaps you could make the build dependent on CMake variable "ENABLE_CLICKHOUSE_BENCHMARK"? If so, it would be nice if the libhdfs3 include directories (l. 119ff) could also be PRIVATE/PUBLIC depending on ENABLE_CLICKHOUSE_BENCHMARK.

Reply:
That sounds good.